packages/debugger/lib/trace/sagas/index.js

```diff
@@ -70,18 +70,37 @@ export function* signalTickSagaCompletion() {
 export function* processTrace(steps) {
   yield put(actions.saveSteps(steps));
 
-  let addresses = [
+  let callAddresses = [
     ...new Set(
       steps
-        .map(
-          ({ op, stack }) =>
-            isCallMnemonic(op)
-              ? //if it's a call, just fetch the address off the stack
-                Codec.Evm.Utils.toAddress(stack[stack.length - 2])
-              : //if it's not a call, just return undefined (we've gone back to
-                //skipping creates)
-                undefined
+        .map(({ op, stack }) =>
+          isCallMnemonic(op)
+            ? //if it's a call, just fetch the address off the stack
+              Codec.Evm.Utils.toAddress(stack[stack.length - 2])
+            : //if it's not a call, just return undefined
+              undefined
         )
+        .filter(address => address !== undefined)
```

Comment:
If you don't mind filtering out ALL falsy values you could 
```
  .filter(address);
```

Reply:
I mean, `undefined` is the only falsy value that will come up here; I'm just doing this for explicitness.